### PR TITLE
Qt: 2 New Theme Options

### PR DESCRIPTION
**Pull request by Daisouji**
### Description of Changes
Added 2 New Options,  A Ruby/Dark Theme (Ruby)  and Another allowing usage of .qss (under the main pcsx2 directory)

**Ruby**

C3
Capture2

**QSS**
C4
c5

### Rationale behind Changes
Thought we needed more color options.

### CODE REVIEW

File: pcsx2-qt/MainWindow.cpp [hunk 1]
```diff
@@ -622,7 +622,63 @@ void MainWindow::setStyleFromSettings()
...
+		qApp->setStyleSheet("QToolTip { color: #ffffff; background-color: #2a82da; border: 1px solid white; }");
+	} 
+	else if (theme == "Custom")
+	{
+		
+		//Additional Theme option than loads .qss from main PCSX2 Directory
+		qApp->setStyle(QStyleFactory::create("Fusion"));
+		
+		QString sheet_content;
+		QFile sheets("./custom.qss");
```
Comment (stenzek) [inline pcsx2-qt/MainWindow.cpp:669]: Probably should use a full path here, the current working directory does get changed at runtime (e.g. `QString::fromStdString(Path::Combine(EmuFolders::DataRoot, "custom.qss"))`

File: pcsx2-qt/MainWindow.cpp [hunk 2]
```diff
@@ -622,7 +622,63 @@ void MainWindow::setStyleFromSettings()
...
+		
+		if(sheets.open(QFile::ReadOnly)){
```
Comment (stenzek) [inline pcsx2-qt/MainWindow.cpp:671]: Apply clang-format here.

File: pcsx2-qt/MainWindow.cpp (hunk 1 above)
Comment (Daisouji) [reply pcsx2-qt/MainWindow.cpp:669]: Sorry I'm kinda iffy on pathing, you are saying concat the filename w/ current-working directory is?

Comment (stenzek) [reply pcsx2-qt/MainWindow.cpp:669]: No, use DataRoot like I posted above. (DataRoot points to either the documents directory, or the pcsx2 directory if running in portable mode).

**Comment by refractionpcsx2** [comment]
If you could clang format that file, you have a mix of tabs and spaces and same line brackets (we do new line brackets)

**Review by lightningterror — CHANGES_REQUESTED** [review]
Some commits can be squashed, also named properly with prefixes.

**Comment by Daisouji** [comment]
> Some commits can be squashed, also named properly with prefixes.

Sure what are proper naming conventions, I didn't see anything on them.

**Comment by Mrlinkwii** [comment]
QT: <whatever you did > like what you did with the pr title

**Comment by lightningterror** [comment]
> Sure what are proper naming conventions, I didn't see anything on them.

Since you're changing the Qt gui, example would be:
Qt: Add two new themes, Ruby/Dark Theme (Ruby)...
Qt: Clang format filename...

**Review by JordanTheToaster — APPROVED** [review]

**Review by Mrlinkwii — APPROVED** [review]

**Pull request merged**